locale/en_US/submission.po

```diff
@@ -1903,3 +1903,9 @@ msgstr "The abstract is too long. It shoud be {$limit} words or less. It is curr
 
 msgid "publication.wordCount"
 msgstr "Word Count: {$count}/{$limit}"
+
+msgid "publication.notEditable"
+msgstr "The publication is not editable"
```

Comment:
change the `msgid` to `galley.cantEditPublished` and change the msgstr to:

> This galley can not be edited because it has already been published.

Reply:
I added the `galley.cantEditPublished` locale key and changed it to `ArticleGalleyForm` validate check.

Should we leave the `publication.notEditable` locale key for the `PKPPublicIdentifiersForm` change? The galley specific one is not suitable there I think.

Reply:
I think the `PKPPublicIdentifiersForm` is only used for editing DOIs of galleys in 3.3, so I think it would be appropriate there too.

Reply:
Do we leave `publication.notEditable` locale key in main branch though?

Reply:
@NateWr I removed the `publication.notEditable` locale key from stable-3_3_0 and replaced it's reference with `galley.cantEditPublished` locale key.

Reply:
Great. Remove the locale key in `main` if it is not used.

Reply:
Changed to `main` branch also.